lib/fog/ovirt/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fog
   module Ovirt
-    VERSION = '0.1.2'
+    VERSION = '0.1.3'.freeze
```

Comment:
why the version bump? why freeze?

Reply:
> why the version bump? 

It changes how exception works, so I think it should be a new version, so exception handlers will need to be changed.

> why freeze?

Ruby's constant are mutable by defaults. Moreover, each time you call a constant string, it allocate new instance of that string.
Calling `freeze` does two main things here:
1. Makes the constant immutable (more or less)
2. Use the constant allocation once, rather then allocate new memory each usage.